testsuite/MDAnalysisTests/topology/base.py

```diff
@@ -62,24 +61,16 @@ def test_mandatory_attributes(self, top):
 
     def test_expected_attributes(self, top):
         # Extra attributes as declared in specific implementations
-        for attr in self.expected_attrs+self.guessed_attrs:
+        for attr in self.expected_attrs:
             assert hasattr(top, attr), 'Missing expected attribute: {}'.format(attr)
     
     def test_no_unexpected_attributes(self, top):
         attrs = set(self.expected_attrs
-                    + self.guessed_attrs
                     + mandatory_attrs
                     + ['indices', 'resindices', 'segindices'])
         for attr in top.attrs:
             assert attr.attrname in attrs, 'Unexpected attribute: {}'.format(attr.attrname)
 
-    def test_guessed_attributes(self, top):
```

Comment:
This test needs to be replaced by a test that checks these attributes are part of the universe.

Reply:
I'm working on this now